workflow/scripts/update_population_db.py

```diff
@@ -0,0 +1,63 @@
+import os
+import shutil
+import sys
+
+sys.stderr = open(snakemake.log[0], "w")
+
+import subprocess as sp
+import pysam
+
+db_path = snakemake.output[0]
+
+
+def sample_in_db(group):
+    samples = pysam.VariantFile(db_path).header.samples
+    if group in samples:
+        if len(samples) == 1:
+            return "only"
+        else:
+            return "one"
+    else:
+        return "not"
+
+
+def copy_to_db(bcf_path):
+    shutil.copyfile(bcf_path, db_path)
+
+
+def update_db(bcf_path, group):
+    sp.run(["bcftools", "index", "-f", db_path])
```

Comment:
Not happy about this. `pysam` and `bcftools` require an index file which has to be created after each update.